project-fortis-interfaces/src/routes/AppPage.js

```diff
@@ -146,16 +146,16 @@ export const AppPage = createReactClass({
     const {
       fromDate, toDate, datetimeSelection, timespanType, dataSource, maintopic,
       bbox, zoomLevel, conjunctivetopics, externalsourceid, selectedplace,
-      category
+      category, sharedLink
     } = dataStore;
     const includeCsv = false;
 
     this.getFlux().actions.DASHBOARD.initializeDashboard(category, () => {
       this.getFlux().actions.DASHBOARD.reloadVisualizationState(
         fromDate, toDate, datetimeSelection, timespanType, dataSource, maintopic,
-        bbox, zoomLevel, conjunctivetopics, externalsourceid, includeCsv, selectedplace, () => {
+        bbox, zoomLevel, conjunctivetopics, externalsourceid, includeCsv, selectedplace, sharedLink, () => {
```

Comment:
The interface for this method is this:

https://github.com/CatalystCode/project-fortis/blob/20289a8bbc183da514ed281790cfdeffe484b554/project-fortis-interfaces/src/actions/Dashboard/index.js#L165-L166

As such, the sharedLink variable here should be removed. The `true` as the last argument in this function call is all we need.

This will also remove the need for unpacking the sharedLink variable on line 149 above.

Reply:
@c-w had that in there because if it isn't there I get the error: 

```

Error in ./src/actions/Dashboard/index.js

/app/src/actions/Dashboard/index.js
  183:112  error  'sharedLink' is not defined  no-undef
```

Is there something else I can do to circumvent the error at all?

Reply:
I mean, I removed it from AppPage okay, but if I remove it from Dashboard I get the no-undef variable.

Reply:
Removal from AppPage is all I mean. Thanks.